README.md

```diff
@@ -46,7 +46,7 @@ The following snippet defines a Kubewarden Policy based on the
 policy:
 
 ```yaml
-apiVersion: kubewarden.suse.com/v1alpha1
+apiVersion: kubewarden.io/v1alpha1
```

Comment:
It's common for the group to be of the form `subdomain.domain` for external API types, not `domain` only. I don't know if `domain` works or if there's somewhere on the machinery where a subdomain is needed. Thing is I have only seen subdomains, never domains directly.

Reply:
My comment applies here and to all instances of this change, we could use `policy.kubewarden.io/v1alpha1`, for example.

Reply:
good point, let me change that to `policies.kubewarden.io`. I'll update the PR